assets/js/components/settings/SettingsActiveModule/Footer.js

```diff
@@ -95,7 +112,10 @@ export default function Footer( props ) {
 		);
 		await clearErrors();
 		history.push( `/connected-services/${ slug }` );
-	}, [ clearErrors, history, viewContext, slug ] );
+		if ( slug === 'analytics' ) {
```

Comment:
It would be a little nice to add a one-line space around the `if` statement to have a little separation.